src/Metadata/class-category-builder.php

```diff
@@ -39,7 +39,10 @@ public function get_metadata(): array {
 	 * @since 3.4.0
 	 */
 	private function build_headline(): void {
-		$category                   = get_queried_object();
-		$this->metadata['headline'] = $this->clean_value( $category->name );
```

Comment:
I think we shouldn't have default value assigned to it because of [this condition](https://github.com/Parsely/wp-parsely/blob/fix/phpstan-errors/src/UI/class-metadata-renderer.php#L110) which decides whether something went wrong or not.